microraiden/microraiden/requests/__init__.py

```diff
@@ -3,15 +3,16 @@
 from requests import Response
 from web3 import Web3
 
-from microraiden import DefaultHTTPClient, Client
+from microraiden import DefaultHTTPClient, Client, HTTPClient
 from microraiden.config import CHANNEL_MANAGER_ADDRESS, CONTRACT_METADATA
 from microraiden.contract_proxy import ChannelContractProxy, ContractProxy
 
-client = None
-http_client = None
+_http_client = None
```

Comment:
So it's not possible to use two different keys in one session? It's probably ok to have a global now, but the library should also support creating multiple `request` contexts.

Reply:
The use of multiple clients/keys is planned for #256 and will follow the design of `requests.Session`.
Currently if you want multiple clients you just have to set up your own `DefaultHTTPClient`s again which is pretty much the same as calling `init()` on the global client.